packages/rspack/src/builtin-plugin/ProgressPlugin.ts

```diff
@@ -1,8 +1,8 @@
 import { RawProgressPluginConfig } from "@rspack/binding";
 import { BuiltinPluginKind, create } from "./base";
 
-export type ProgressPluginOptions = RawProgressPluginConfig | undefined;
+export type ProgressPluginArgument = RawProgressPluginConfig | undefined;
```

Comment:
Do we have to even align with the type name? For consistency I would love to use `XXXOptions`

Reply:
I also prefer `XXXOptions`, but types in webpack are not consistent..., let's keep the type alignment for now, at least for facade type